src/core/DY.Core.Trace.Manipulation.fst

```diff
@@ -547,7 +571,7 @@ val trigger_event: principal -> string -> bytes -> traceful unit
 let trigger_event prin tag content =
   add_entry (Event prin tag content)
 
-#push-options "--z3rlimit 30"
+#push-options "--z3rlimit 25 --ifuel 2"
```

Comment:
I am curious why this needs an increase in ifuel?

Reply:
Ah interesting, I just hit the same flakiness, and although increasing the rlimit works in the interactive mode, it does not work when running `make`, however increasing ifuel works... weird, probably an F* bug

Reply:
In one of the recent merged PRs I put a better fix (this theorem proves an `exists`, so I give a witness rather than letting Z3 guessing it)